packages/frontend-2/components/pricingTable/Plan.vue

```diff
@@ -56,29 +56,45 @@
       </div>
     </div>
     <ul class="flex flex-col gap-y-2 mt-4 pt-3 border-t border-outline-3">
-      <li
+      <PricingTablePlanFeature
```

Comment:
Some of these features a just "cosmetic", only this component cares about rendering, they are not really features so didnt add them to the shared object